web/packages/teleport/src/Player/SshPlayer.tsx

```diff
@@ -53,22 +56,31 @@ export default function Player({ sid, clusterId }) {
     );
   }
 
-  if (!isLoading && eventCount === 0) {
-    return (
-      <StatusBox>
-        <Text typography="h4">
-          Recording for this session is not available.
-        </Text>
-      </StatusBox>
-    );
-  }
-
   return (
     <StyledPlayer>
       <Flex flex="1" flexDirection="column" overflow="auto">
         <Xterm tty={tty} />
       </Flex>
-      {eventCount > 0 && <ProgressBarTty tty={tty} />}
+      <ProgressBar
+        min={0}
+        max={durationMs}
+        current={time}
+        disabled={
+          playerStatus === TtyStatusEnum.ERROR ||
+          playerStatus === TtyStatusEnum.COMPLETE
+        }
```

Comment:
I don't think we should disable when complete. I haven't used it much, but it seems like wanting to move through/restart a completed video would be a valid choice.

Reply:
Yeah but it doesn't work and it would be a fair amount of work to make it work when there's already a replay button that will restart it for us.

Reply:
Fair enough